op_hello_js/lib.rs

```diff
@@ -0,0 +1,27 @@
+extern crate deno;
+use deno::*;
+
+pub fn init(&mut isolate: Isolate) -> Result<(), ErrBox> {
+  isolate.register_op("hello", op_hello); // register_op defined by #3002
+  isolate.execute("hello.js")?;
```

Comment:
I think it should be:
```rust
isolate.execute("hello.js", include_str!("hello.js"));`
```

Reply:
Yeah, probably. I'm a bit concerned about double counting the source code if we do snapshots...